### ISSUE TITLE
Submission systems bug

### ISSUE THREAD

**Issue by tommycarpi** [recsyschallenge/2016#19]
There is a bug in the submission system, basically if you use too verbose _**labels**_ the system returns a _**generic error**_ saying that something went wrong. <br>
I simply reduced the text of label and the same submission was accepted. <br>

Please fix this or set a char limit in the textfield to prevent it :) .

**Comment by desertnaut** [comment]
Good point - it caused some headache to us, too! Thanks for pointing it out...
